GPSTest/src/main/java/com/chesapeaketechnology/gnssmonkey/AGpsMonkeyActivity.java

```diff
@@ -149,11 +184,37 @@ protected void onStop() {
         }
     }
 
+    /**
+     * According to the Android documentation, there are situations where the system will kill the
```

Comment:
I will work on this for the Network Survey app and let you know what I figure out.

Reply:
Eliza and I discussed this yesterday and we decided that it makes the most sense to stop the service in onDestroy(). Yes, there are cases where that may not be called, but the documentation doesn't say what they are. I'll push an update shortly with those changes.